packages/launcher/src/renderer/ModalView.js

```diff
@@ -0,0 +1,40 @@
+//@flow
+
+import React, {
+  createRef,
+  Component,
+  type Element,
+  type ElementRef,
+} from 'react'
+import { View, StyleSheet, Text } from 'react-native-web'
+import ReactModal from 'react-modal'
+
+type Props = {
+  onRequestClose: () => void,
+  children?: ?Element<any>,
+}
+
+export default class ModalView extends Component<Props> {
+  render() {
+    return (
+      <ReactModal
+        isOpen={true}
+        onRequestClose={this.props.onRequestClose}
+        {...this.props}>
+        <View style={styles.container}>{this.props.children}</View>
+      </ReactModal>
+    )
+  }
+}
+
+const COLOR_WHITE = '#ffffff'
```

Comment:
Not an issue but could you gather all the colors and other shared styles in a single module as we've been doing with Mainframe "classic" please?